HW_5/Task1.py

```diff
@@ -0,0 +1,12 @@
+# Напишите функцию, которая принимает на вход строку - абсолютный путь
+# до файла. Функция возвращает кортеж из трёх элементов: путь, имя файла,
+# расширение файла.
+
+def path_name_extension(path: str) -> tuple:
+    path, name = path.rsplit('\\', 1)
+    name, extension = name.split('.')
+    return path, name, extension
```

Comment:
Можно чуть сократить.
```suggestion
    *path, name= path.rsplit('\\', 1)
    return *path, *name.split('.')
```